addons/sys_gestures/XEH_postInit.sqf

```diff
@@ -0,0 +1,66 @@
+#include "script_component.hpp"
+if (is3DEN || {!hasInterface || {!(GVAR(gesturesEnabled))}}) exitWith {};
+if (!isClass (configFile >> "CfgPatches" >> "ace_common")) exitWith {}; // No ACE exit
+
+GVAR(vestRadioArr) = call compile (GVAR(vestRadios));
+GVAR(headsetRadioArr) = call compile (GVAR(headsetRadios));
+
+["acre_startedSpeaking", {
+    params ["_unit", "_onRadio", "_radio"];
+
+    if (!_onRadio ||
+        { !isNull objectParent _unit } ||
+        { !(cameraView in ["INTERNAL","EXTERNAL"]) } ||
+        { ace_common_isReloading } ||
+        { isWeaponDeployed _unit } ||
+        { animationState _unit in GVAR(blackListAnims) } ||
+        { currentWeapon _unit in GVAR(binoClasses) } ) exitWith {};
+
+    private _hasVest = vest _unit != "";
+    private _hasHeadgear = headgear _unit != "";
+    if (!_hasVest && !_hasHeadgear) exitWith {};
+
+    private _baseRadio = _radio call acre_api_fnc_getBaseRadio;
+    private _isVestRadio = (GVAR(vestRadioArr) findIf {_x isEqualTo _baseRadio}) != -1;
+    private _isHeadsetRadio = (GVAR(headsetRadioArr) findIf {_x isEqualTo _baseRadio}) != -1;
```

Comment:
`in` is much much faster. (depending on amount of array elements)

```sqf
// 0.0028 ms
_arr = ["a", "b", "c"];
_item = "a";
_arr findIf {_x isEqualTo _item} != -1

// 0.0043 ms
_arr = ["a", "b", "c"];
_item = "c";
_arr findIf {_x isEqualTo _item} != -1
```

```sqf
// 0.0014 ms
_arr = ["a", "b", "c"];
_item = "a";
_item in _arr

// 0.0014 ms
_arr = ["a", "b", "c"];
_item = "c";
_item in _arr
```

Reply:
Yeah, this seems obvious, not sure why I switched to `findIf`... Fixed in latest commit.